go/service/config.go

```diff
@@ -46,9 +47,37 @@ func (h ConfigHandler) GetCurrentStatus(_ context.Context, sessionID int) (res k
 	return
 }
 
-func getPlatformInfo() keybase1.PlatformInfo {
+func (h ConfigHandler) getOsVersion() string {
+	var cmd string
+	var args []string
+	var result []byte
+	var err error
+
+	if runtime.GOOS == "windows" {
+		cmd = "cmd"
+		args = []string{"/c", "ver"}
+	} else if runtime.GOOS == "darwin" {
+		cmd = "/usr/bin/sw_vers"
```

Comment:
It would be nice to also have `sw_vers -buildVersion` too but would require doing 2 execs.

And if so, would be good to return 2 strings: 
```go
func (h ConfigHandler) osVersion() (string, string) {
```